File: lavugio-back/src/main/java/com/backend/lavugio/dto/ride/RideDestinationDTO.java
package com.backend.lavugio.dto.ride;

import jakarta.persistence.Column;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Data
public class RideDestinationDTO {
    private StopBaseDTO location;
    private String address;

    private String streetName;
    private String city;
    private String country;
    private int streetNumber;
    private int zipCode;
}

File: lavugio-front/src/app/shared/models/ride/rideDestinationDTO.ts
export interface StopBaseDTO {
  orderIndex: number;
  latitude: number;
  longitude: number;
}

export interface RideDestinationDTO {
  location: StopBaseDTO;
  address: string;
  streetName: string;
  city: string;
  country: string;
  streetNumber: number;
  zipCode: number;
}
